typedoc.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Should this also have "entrypoints"? (like in #2518)

Reply:
So I was basing this on the arguments directly passed to `yarn run typedoc` in `package.json`.

And that only is `--includes src/` which would refer to https://typedoc.org/guides/options/#includes in the documentation. Which we don't use.

`--entryPoints` seems (from my reading of the docs) to be the way to specify multiple files (or folders) as the starting points for the docs.

But we only want to document the extension API, iirc, and not the whole of `src`. So I don't think we do.

Reply:
Did you test how the documentation looks like? In my PR I had to use entrypoints for proper generation (https://typedoc.org/guides/options/#entrypoints)

Reply:
Yeah I did, it looked fine. I'll post some screenshots for you to look over.